scripts/version.js

```diff
@@ -3,7 +3,7 @@ const glob = require('glob');
 const path = require('path');
 const execSync = require('child_process').execSync;
 const PACKAGE = require('../package.json');
-const dirs = glob.sync('./example/*', {});
+const dirs = glob.sync('./examples/*', {});
```

Comment:
I don't think this would totally work, it would get all the examples in the advanced directory. We could replace that line and the rest of the file below with:
```javascript
const packageJSONs = glob.sync('./examples/**/package.json', {});
packageJSONs.forEach(function(filePath) {
  let pkg = fs.readJsonSync( filePath );
  pkg.devDependencies[PACKAGE.name] = PACKAGE.version;
  fs.writeFileSync(filePath, JSON.stringify(pkg, null, 2));
  // Add the package.json file to staging so it'll get commited
  execSync(`git add ${filePath}`);
});
```

Reply:
done.